package.json

```diff
@@ -49,38 +49,38 @@
         "rollup-plugin-visualizer": "4.0.4",
         "stylus": "0.55.0",
         "ts-jest": "27.0.3",
-        "typescript": "4.1.2",
+        "typescript": "4.1.6",
         "whatwg-fetch": "3.5.0"
     },
     "dependencies": {
         "@badrap/result": "0.2.6",
-        "@capacitor-community/keep-awake": "3.0.0",
-        "@capacitor/android": "^4.0.0",
-        "@capacitor/app": "^4.0.0",
-        "@capacitor/browser": "^4.0.0",
-        "@capacitor/clipboard": "^4.0.0",
-        "@capacitor/core": "^4.0.0",
-        "@capacitor/device": "^4.0.0",
-        "@capacitor/dialog": "^4.0.0",
-        "@capacitor/filesystem": "^4.0.0",
-        "@capacitor/haptics": "^4.0.0",
-        "@capacitor/ios": "^4.0.0",
-        "@capacitor/keyboard": "^4.0.0",
-        "@capacitor/network": "^4.0.0",
-        "@capacitor/preferences": "4.0.2",
-        "@capacitor/push-notifications": "^4.0.0",
-        "@capacitor/share": "^4.0.0",
-        "@capacitor/splash-screen": "^4.0.0",
-        "@capacitor/status-bar": "^4.0.0",
-        "@capacitor/toast": "^4.0.0",
+        "@capacitor-community/keep-awake": "4.0.0",
+        "@capacitor/android": "^5.0.0",
+        "@capacitor/app": "^5.0.0",
+        "@capacitor/browser": "^5.0.0",
+        "@capacitor/clipboard": "^5.0.0",
+        "@capacitor/core": "^5.0.0",
+        "@capacitor/device": "^5.0.0",
+        "@capacitor/dialog": "^5.0.0",
+        "@capacitor/filesystem": "^5.0.0",
+        "@capacitor/haptics": "^5.0.0",
+        "@capacitor/ios": "^5.0.0",
+        "@capacitor/keyboard": "^5.0.0",
+        "@capacitor/network": "^5.0.0",
+        "@capacitor/preferences": "^5.0.0",
+        "@capacitor/push-notifications": "^5.0.0",
+        "@capacitor/share": "^5.0.0",
+        "@capacitor/splash-screen": "^5.0.0",
+        "@capacitor/status-bar": "^5.0.0",
+        "@capacitor/toast": "^5.0.0",
         "@types/d3": "4.5.0",
         "@types/lodash-es": "4.17.3",
-        "capacitor-sound-effect": "3.0.0",
-        "capacitor-stockfish": "4.0.0",
-        "capacitor-stockfish-variants": "2.0.0",
+        "capacitor-sound-effect": "https://github.com/veloce/capacitor-sound-effect",
+        "capacitor-stockfish": "https://github.com/vovagorodok/capacitor-stockfish",
+        "capacitor-stockfish-variants": "https://github.com/vovagorodok/capacitor-stockfish-variants",
```

Comment:
This capacitors are upgraded and wait for release:
- `capacitor-sound-effect` already upgraded  and wait for release: https://github.com/veloce/capacitor-sound-effect/issues/15
- `capacitor-stockfish` already upgraded  and wait for release: https://github.com/veloce/capacitor-stockfish/pull/11
- `capacitor-stockfish-variants` wait for merge upgrade and release: https://github.com/veloce/capacitor-stockfish-variants/pull/1